newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/webservices/JakartaWebServiceVisitor.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
Should I consider changing the package structure here as well and put these two classes in a `javax` and `jakarta` namespace?

Also, just noticed the copyright is not right here. Probably in a few other places as well. I'll dbl check that.

Reply:
Perhaps if that is the pattern you are following else where

Reply:
but honestly if its just one class and the other class is prefixed "Javax...", may be overkill for new packages